geemusic/__init__.py

```diff
@@ -10,10 +10,15 @@
 app = Flask(__name__)
 ask = Ask(app, '/alexa')
 
+def lambda_handler(event, _context):
+    return ask.run_aws_lambda(event)
+
 if str(environ['DEBUG_MODE']) is True:
-    log_level = logging.DEBUG
+	log_level = logging.DEBUG
```

Comment:
switch tabs to 4 spaces (might want to do this throughout the entire PR)

Reply:
I ran `find . -type f -exec sed -i.orig 's/\t/    /g' {} +` everywhere and this was the only file with tabs still in it. Feel free to correct me if I'm wrong